### PR TITLE
German translation for driver requirements

### PR DESCRIPTION
**Pull request by boris779**
### Description
German translation for driver requirements, also changed one german header

### Motivation and Context
Missing german translation for that page

### Types of changes
- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist
- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/webdriver/driver_requirements.de.md [hunk 1]
```diff
@@ -384,19 +373,19 @@ System.setProperty("webdriver.edge.driver", "C:/path/to/MicrosoftWebDriver.exe")
...
+weiter unterstützt.
 
-While the Selenium project provides binaries for both the 32-bit and 64-bit 
-versions of Internet Explorer, there are some 
-[limitations](//jimevansmusic.blogspot.co.uk/2014/09/screenshots-sendkeys-and-sixty-four.html)
-with Internet Explorer 10 & 11 with the 64-bit driver, but using the 32-bit 
-driver continues to work well. It should be noted that as Internet Explorer
-preferences are saved against the logged in user's account, some 
-[additional setup is required](//github.com/SeleniumHQ/selenium/wiki/InternetExplorerDriver#required-configuration).
+Das Seleniumprojekt stellt die Driver für 32-bit und 64-bit Versionen des Internet
```
Comment (harsha509) [inline docs_source_files/content/webdriver/driver_requirements.de.md]: Selenium projekt

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @boris779 !

Thanks for the PR.

Requested for a small change! can you please look into it

File: docs_source_files/content/webdriver/driver_requirements.de.md (hunk 1 above)
Comment (boris779) [reply docs_source_files/content/webdriver/driver_requirements.de.md]: "Selenium-Projekt" would be the best correct ssolution.
I will update my pull request shortly

**Review by harsha509 — APPROVED** [review]
Thank you @boris779 !

**Pull request merged**